src/UartLibFramesDefines.cpp

```diff
@@ -0,0 +1,14 @@
+/**
+ * @file UartLibFramesDefines.cpp
+ * @author Maciej Kurcius
+ * @brief 
+ * @version 0.1
+ * @date 2022-03-10
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+
+#include "UartLib.h"
+
+
```

Comment:
For now, is not. But in the future, when we would have a lot of pre-definied frames to send by digital board, this frame should be define in this file because, frames will be separate from uart library. So we can delete it and add when we will need this option.